### DISCUSSION TITLE
Where to communicate?

### DISCUSSION THREAD

**Discussion by GarthDB** [DesignSystemsEngineer/designsystemsengineer.github.io#40] [Polls]
To start, we are using GitHub because it feels like a natural tool for Front End/Design/Design System *Engineering*, but would we benefit from real-time - ish chatting?

GitHub doesn't seem ideal long-term for all communication, but there are so many other platforms it's hard to find information sometimes. Slack/Discord are great, but it can be hard to be in so many.

Ideally, we'd love to find a solution for discussions with a low entry barrier and fully inclusive of people who want to participate respectfully.

Let me know if I've missed any other options.

**Comment by break-stuff** [comment]
I regularly check Discord, but I don't really peruse GitHub discussions the same way.
